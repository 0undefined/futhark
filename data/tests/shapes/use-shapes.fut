// Test that a variable shape annotation is actually bound.

fun [int] main([int,n] a) =
  map(+n, a)
