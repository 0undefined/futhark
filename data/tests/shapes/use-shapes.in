[42,1337,5,4,3,2,1]
