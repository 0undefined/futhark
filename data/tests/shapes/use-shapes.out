[49,1344,12,11,10,9,8]
